frontend_tests/node_tests/activity.js

```diff
@@ -92,15 +102,71 @@ activity.update_huddles = function () {};
 global.compile_template('user_presence_row');
 global.compile_template('user_presence_rows');
 
+var presence_info = {};
+presence_info[alice.user_id] = { status: 'inactive' };
+presence_info[fred.user_id] = { status: 'active' };
+presence_info[jill.user_id] = { status: 'active' };
+
+presence.presence_info = presence_info;
+
+(function test_get_status() {
+    assert.equal(presence.get_status(page_params.user_id), "active");
+    assert.equal(presence.get_status(alice.user_id), "inactive");
+    assert.equal(presence.get_status(fred.user_id), "active");
+}());
+
+(function test_last_seen_status_from_date() {
+    // base_date set to March 1 2016 12.30 AM
+    var base_date = new XDate(2016, 2, 1, 0, 30);
```

Comment:
@showell No. It's 0 indexed. Even I was surprised at first and took me some time to figure out why the output of tests are off by month when I used 3 instead of 2 :smile: 

![screenshot from 2017-05-10 23 25 47](https://cloud.githubusercontent.com/assets/7190633/25913424/46b17a78-35d8-11e7-86b2-b3c690bdb3a1.png)